00XX-common-ledger-protocol/00XX-common-ledger-protocol.md

```diff
@@ -0,0 +1,262 @@
+# Common Ledger Protocol (CLP)
+
+## Preface
+
+This document describes the Common Ledger Protocol (CLP), a ledger protocol for
+bilateral liquidity relationships. It is similar in function to [Plugin RPC],
+but has been rewritten to use OER instead of JSON, and extensibility features
+have been added.
+
+## Introduction
+
+### Motivation
+
+There are many different types of ledgers, and Interledger aims to interoperate
+all of them. For fast ledgers that support hashlocks, it is enough to wrap the
+ledger's API in a [Ledger Plugin]. These allow you to send interledger payments
+through anyone else on that ledger who is running your plugin.
+
+In lots of scenarios, we don't have an underlying ledger that's fast enough to
+do every ILP payment on-ledger. There are also many situations when two parties
+trust one another up to an amount, and don't need any third-party ledger to
+book-keep for them. Currently, this need is filled by [ILP Plugin Virtual],
+which uses [Plugin RPC] as its ledger protocol. JSON messages are passed back
+and forth to perform transfers and send quote requests. Custom JSON messages
+can be added to extend the Plugin RPC messages, but they aren't well defined.
+
+Because the use case for a bilateral ledger protocols is so ubiquitous, the CLP
+has been invented to make the protocol extensible, efficient, and friendly to
+re-implementation.
+
+### Scope
+
+The CLP, despite its name, is just another ledger protocol. It manages
```

Comment:
This makes it sound like a really inappropriate name. The reason it's called that is 1) because we think that it will be very widespread due to the fact that trustlines are such a useful way of modeling accounts in an ILP setup and 2) because it is made up of pieces that can also be common to a lot of other ledger (plugin) protocols (like all of the bilateral payment channel ones).

Reply:
I agree with Evan; we discussed this in the research team meeting yesterday. Sure, other protocols, like the existing RPC for instance, can do the same as CLP, but we think CLP is the best one, and we want people to use it whenever they implement Interledger from scratch, right? Or are you saying we don't care about that?

We also don't say in the ILQP spec that it's "just another quoting protocol".